__tests__/__utils__/testAttributeHandler.js

```diff
@@ -81,5 +81,5 @@ global.testAttribute = (constructor, listChartOptionMock, eventName) => {
 
 global.flushPromises = () => {
   // eslint-disable-next-line no-undef
-  return new Promise((resolve) => setImmediate(resolve));
```

Comment:
FYI @scolladon 
Even if setImmediate() and process.nextTick don't behave exactly the same, I think that both of them are fine for us, because of how the reactivity manager handles the tasks enqueued.
Anyway, if you think that we should better keep the setImmediate() logic, we can have a look to polyfills or other alternatives

Reply:
Let's go with `process.nextTick` !